src/script/page/AppLock.tsx

```diff
@@ -77,9 +77,10 @@ const AppLock: React.FC<AppLockProps> = ({
   const [setupPassphrase, setSetupPassphrase] = useState('');
   const [inactivityTimeoutId, setInactivityTimeoutId] = useState<number>();
   const [scheduledTimeoutId, setScheduledTimeoutId] = useState<number>();
-  const {isAppLockActivated, isAppLockEnabled} = useKoSubscribableChildren(appLockState, [
+  const {isAppLockActivated, isAppLockEnabled, isAppLockEnforced} = useKoSubscribableChildren(appLockState, [
     'isAppLockActivated',
     'isAppLockEnabled',
+    'isAppLockEnabled',
```

Comment:
Shouldn’t this be isAppLockEnforced?
```suggestion
    'isAppLockEnforced',
```

Reply:
Good catch, thanks :)